component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/mepin/MepinTransactions.java

```diff
@@ -138,6 +147,15 @@ protected String postRequest(String url, String query, String username, String p
         return responseString;
     }
 
+    private String getRandomHexString() {

```

Comment:
Can you add a method level comment

Reply:
fixed with 8b37d635bec44c8d19a6cebc6fed7eaaec0ff81c